Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "commands"
+require "cli/parser"
+require "json"
+require "net/http"
+require "open-uri"
+
+module Homebrew
+  module_function
+
+  SPDX_FOLDER_PATH = (HOMEBREW_LIBRARY_PATH/"data").freeze
+  FILE_NAME = "spdx.json"
+  SPDX_DATA_URL = "https://raw.githubusercontent.com/spdx/license-list-data/HEAD/json/licenses.json"
+
+  def update_license_data_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update_license_data` <cmd>
+
+          Update SPDX license data in the Homebrew repository.
+      EOS
+      switch "--fail-if-changed",
+             description: "Return a failing status code if current license data's version is different from"\
+                           "the upstream. This can be used to notify CI when the SPDX license data is out of date."
```

Comment:
```suggestion
             description: "Return a failing status code if current license data's version is different from " \
                          "the upstream. This can be used to notify CI when the SPDX license data is out of date."
```